appender-core/src/main/java/com/van/logging/aws/S3Configuration.java

```diff
@@ -161,8 +160,14 @@ static Region resolveRegion(String str) {
     }
 
     public String toString() {
-        return String.format("S3 configuration (%s:%s in region %s; compressed: %s)",
-            this.bucket, this.path, this.region, this.compressionEnabled
-        );
+        if (this.path != null && !this.path.isEmpty()) {
```

Comment:
Try using `StringUtils.isTruthy(this.path)`?

https://github.com/bluedenim/log4j-s3-search/blob/master/appender-core/src/main/java/com/van/logging/utils/StringUtils.java#L15

Reply:
Sure. Incorporated the change